nilearn/input_data/tests/test_nifti_labels_masker.py

```diff
@@ -11,8 +11,7 @@
 import nibabel
 
 from nilearn.input_data.nifti_labels_masker import NiftiLabelsMasker
-from nilearn._utils import testing
-from nilearn._utils import as_ndarray
+from nilearn._utils import testing, as_ndarray, DimensionError
```

Comment:
You are still importing DimensionError from nilearn._utils rather than nilearn._utils.exceptions